src/glint.gleam

```diff
@@ -871,13 +912,19 @@ fn flag_help_to_string_with_description(help: FlagHelp) -> String {
 fn subcommands_help_to_string(help: List(Metadata), config: Config) -> String {
   use <- bool.guard(help == [], "")
 
+  let longest_subcommand_length =
+    help
+    |> list.map(fn(h) { h.name })
+    |> utils.max_string_length
+    |> int.max(min_column_width)
+
   case config.pretty_help {
     None -> subcommands_heading
     Some(pretty) -> heading_style(subcommands_heading, pretty.subcommands)
   }
   <> {
     help
-    |> list.map(subcommand_help_to_string)
+    |> list.map(subcommand_help_to_string(_, longest_subcommand_length + 2))
```

Comment:
❓ Where does this 2 come from?

Reply:
Since this 2-space indent occurs in a few places, I think it's worth pulling it out as a constant.

Reply:
Pulled out to config.